Calibration/EcalCalibAlgos/src/ECALpedestalPCLHarvester.cc

```diff
@@ -155,16 +155,13 @@ void ECALpedestalPCLHarvester::fillDescriptions(edm::ConfigurationDescriptions&
 }
 
 void ECALpedestalPCLHarvester::endRun(edm::Run const& run, edm::EventSetup const& isetup) {
-  edm::ESHandle<EcalChannelStatus> chStatus;
-  isetup.get<EcalChannelStatusRcd>().get(chStatus);
+  const auto chStatus = isetup.getHandle(channelsStatusToken_);
```

Comment:
why not use getData here as well?

Reply:
Also here `channelStatus_` is a pointer. How would you solve it?

Reply:
```suggestion
  channelStatus_ = &isetup.getData(channelsStatusToken_);
```